quinn-udp/tests/tests.rs

```diff
@@ -186,6 +186,36 @@ fn gso() {
     );
 }
 
+#[test]
+fn socket_buffers() {
```

Comment:
Could this test also check that the send and recv buffer size was updated via [`socket2` `send_buffer_size`](https://docs.rs/socket2/latest/socket2/struct.Socket.html#method.send_buffer_size) etc.?